### PR TITLE
Add Victor's article

### PR DESCRIPTION
**Pull request by victor-cody**

### CODE REVIEW

**Comment by kaf-lamed-beyt** [comment]
To think that I was almost going to ask you if I can place this here, under your tweet. 

Thanks for contributing, Vick! 😄

File: src/utils/articles.js [hunk 1]
```diff
@@ -392,10 +392,21 @@ export const twentyTwo = [
...
     entry_id: 20,
   },
+  {
+    title: 'My 2022 Year in Review: Stepping into Growth',
+    previewText:
+      'My personal reflections for 2022. TL;DR: This was a year of personal growth. I didn’t plan how long this review would turn out, because 2022 was eventful. I do hope it turns out to be worthwhile',
+    author: 'Victor Ikechukwu',
+    readMore:
+      'https://vickyikechukwu.hashnode.dev/my-2022-year-in-review-stepping-into-growth',
+    date: new Date('Feb 27 2023, 23:05:00'),
```
Comment (kaf-lamed-beyt) [inline src/utils/articles.js:410]: ..and I think you're missing the `entry_id` key below this line 🤭

**Review by kaf-lamed-beyt — CHANGES_REQUESTED** [review]

File: src/utils/articles.js (hunk 1 above)
Comment (kaf-lamed-beyt) [reply src/utils/articles.js:410]: I've gone ahead to add the property, and merged the PR!

Thanks again, chief! 😎

**Comment by victor-cody** [comment]
> To think that I was almost going to ask you if I can place this here, under your tweet.
> 
> Thanks for contributing, Vick! 😄

Hahaha 😂😂😂 sorry for the late reply. Lol, you're attaching too much importance to this, it should've been me asking if i could place it here 🙏🏼🫡

File: src/utils/articles.js (hunk 1 above)
Comment (victor-cody) [reply src/utils/articles.js:410]: Ahhan, ahhan, you are too fast my boss. Thanks for everything bigger chief 🫡🫡🧎🏽‍♂️

**Comment by kaf-lamed-beyt** [comment]
> Hahaha joyjoyjoy sorry for the late reply. Lol, you're attaching too much importance to this, it should've been me asking if i could place it here 🙏🏼🫡

Lmao! it's all good! 🤣 😂 😂

**Pull request merged**